<launch>
    <arg name="robot" value="maggie" />

    <group ns="$(arg robot)">
        <rosparam file="$(find maggie_navigation_config)/config/initial_pose.yaml" ns="initial_pose" />

        <node pkg="maggie_navigation_config" name="initial_status_sender" type="initial_status_sender" output="screen" />
    </group>
</launch>
